### ISSUE TITLE
`babel-make-styles` plugin causing build failures after migrating react-avatar to new DX

### ISSUE THREAD

**Issue by Hotell** [microsoft/fluentui#18903]
After migrating `react-avatar` to new DX via `yarn nx workspace-generator migrate-converged-pkg` build started failing

> https://github.com/microsoft/fluentui/pull/18866
---

The CI file is caused by `babel-make-styles` plugin:

**Current:**

![2021-07-12 at 13 51](https://user-images.githubusercontent.com/1223799/125283016-49189480-e318-11eb-8e33-76a321a31e75.png)

**After removing make styles plugin from babelrc:**
![2021-07-12 at 13 56](https://user-images.githubusercontent.com/1223799/125283803-0905e180-e319-11eb-8410-826decc849f7.png)

↓↓↓↓

`yarn lage build --to @fluentui/react-avatar`

↓↓↓↓

![2021-07-12 at 13 52](https://user-images.githubusercontent.com/1223799/125283096-60578200-e318-11eb-9669-f7466c115858.png)

can you help us out @layershifter  ?

---

there is also another issue, that is caused by `make-styles` - it uses ES2015+ language features, thus API extractor will properly notify us that something is not ok. (this was reported/discovered by @ling1726 )

![2021-07-12 at 13 53](https://user-images.githubusercontent.com/1223799/125283272-8da43000-e318-11eb-8f33-678c37c92df1.png)

You'll need to change lib to `es2015` 
![2021-07-12 at 13 53](https://user-images.githubusercontent.com/1223799/125283388-ac0a2b80-e318-11eb-84c8-1402ec65386d.png)

_Originally posted by @Hotell in https://github.com/microsoft/fluentui/pull/18866#issuecomment-878216124_

**Comment by Hotell** [comment]
quick fix provided by @layershifter 


This looks like an issue with `@linaria/shaker`, I will debug the problem and report it. Currently, to unblock the PR I suggest to apply the following change:

```diff
  const rootClasses = styles.root;
  const svgClasses = styles.svg;

  return { containerProps, nativeProps, rootClasses, svgClasses };
};

+export const renderIcon = (
-const renderIcon = (
  SVGElement: (props: { svgClasses: string }) => JSX.Element,
): React.FC<React.HTMLAttributes<HTMLSpanElement>> => props => {
  const { containerProps, nativeProps, rootClasses, svgClasses } = useIconProps(props);
```

**Comment by ling1726** [comment]
I managed to get a minimal repro thanks to debugging guidance from @layershifter. It seems that this is an issue with linaria shaker.

Submitted issue: #804 to them with the minimal repro

**Comment by Hotell** [comment]
@layershifter @ling1726 I see this issue was closed https://github.com/callstack/linaria/issues/804, does it mean this one could be closed as well?

**Comment by layershifter** [comment]
@Hotell yes, this was fixed and can be closed.
